libs/src/api/user.js

```diff
@@ -283,12 +283,12 @@ class Users extends Base {
       didMetadataUpdate = true
     }
 
-    if (didMetadataUpdate) {
-      await this.updateAndUploadMetadata({
-        newMetadata: metadata,
-        userId: metadata.user_id
-      })
-    }
+    // if (didMetadataUpdate) {
+    //   await this.updateAndUploadMetadata({
+    //     newMetadata: metadata,
```

Comment:
even for a DO NOT MERGE branch, let's revert all of these unrelated changes - it's better to keep our code diff as clean as possible